cli/src/config/hints.toml

```diff
@@ -0,0 +1,2 @@
+[hints]
+howto-resolve-conflicts = true
```

Comment:
nit: maybe it should be `how-to-`?

Reply:
I did think about that. FWIW my reasoning was that I was going for 'how-to' (i.e. a how-to on resolving conflicts) rather than 'how to' and given it's already hyphenated I dropped the hyphen. I'm happy to add it.

Perhaps just `hints.resolving-conflicts` is better though?

Reply:
I've dropped the 'how to' part - I suppose there's quite a good chance that it would be a prefix on all of the hints since a lot of hints are going to point out how to do something so I'm not sure it adds anything.